data-transformers/transformers/propertiesTransformer.py

```diff
@@ -189,74 +192,72 @@ def createVersionDict(a):
             continue
     return d
 
+def getFromMinio(url):
+    from commons import getFromUrl
+    return getFromUrl(url)
+
+def hostNeedsSaving(infoData, cassandraKeyspace):
+    hostID = ""
+    
+    ob = json.loads(infoData.decode())
+    for e in ob:
+        e = e.split("=")
+        if e[0] == "ID":
+            hostID = e[1]
+            break
+    
+    if hostID == "":
+        return false
+    
+    hostNotSaved = sc.cassandraTable(cassandraKeyspace, "host_properties") \
+        .select("host_id") \
+        .where("host_id=?", hostID) \
+        .isEmpty()
+        
+    return hostNotSaved
+
 def main():
     from pyspark_cassandra import CassandraSparkContext
     from pyspark import SparkConf
+    from pyspark import SparkFiles
     
     # Takes arguments
-    sparkMaster = sys.argv[1]
-    cassandraHost = sys.argv[2]
-    minioHost = sys.argv[3]
-    filePath = sys.argv[4]
-    trialID = sys.argv[5]
-    experimentID = trialID.split("_")[0]
-    containerID = sys.argv[7]
-    cassandraKeyspace = "benchflow"
+    minioHost = sys.argv[1]
+    filePath = sys.argv[2]
+    trialID = sys.argv[3]
+    experimentID = sys.argv[4]
+    SUTName = sys.argv[5]
+    containerID = sys.argv[6]
     table = "container_properties"
-    minioPort = "9000"
     
     # Set configuration for spark context
-    conf = SparkConf() \
-        .setAppName("Properties Transformer") \
-        .setMaster(sparkMaster) \
-        .set("spark.cassandra.connection.host", cassandraHost)
+    conf = SparkConf().setAppName("Properties Transformer")
     sc = CassandraSparkContext(conf=conf)
     
-    res = urllib2.urlopen("http://"+minioHost+":"+minioPort+"/"+filePath+"_inspect.gz")
-    compressed = io.BytesIO(res.read())
-    decompressed = gzip.GzipFile(fileobj=compressed)
-    inspectData = decompressed.readline()
+    # Retrieve the configuration file that was sent to spark
```

Comment:
I guess this is a common function to multiple transformers. Are you going to refactor it in this pull request or in a new one? If it's a new one, open an issue about this and point to this comment